components/mediation-connector/org.wso2.carbon.mediation.connector.as4/pom.xml

```diff
@@ -152,11 +152,33 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.0</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>1.7</source>
```

Comment:
Not sure about dependency management. 
Also, isn't all our source versions in 1.8 now?